client/src/components/public/Projects.jsx

```diff
@@ -92,7 +108,11 @@ class _Projects extends Component {
 
     if (countResponse.isSuccessful) {
       const counts = (await countResponse.json()).counts;
-      this.setState({ counts });
+      let noPages = {};
+      Object.keys(counts).forEach(key => {
```

Comment:
We can use `reduce` instead of `forEach` here to construct the `numPages` object